.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+name: Generate artifacts
+
+on:
+  push:
+    paths:
+      - 'src/**.md'
+  workflow_dispatch:
+
+jobs:
+  generateFiles:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: make install
+      - run: make build
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v4.2.3
+        with:
+          title: 'Automatic generation of artifacts'
+          body: 'New files have been generated'
```

Comment:
Could it be possible for the PR body to list?

- The commit ID
- The description of the commits

Reply:
it can with a simple git status into a variable but it will always list the exact same thing aka, the names of the 3 files in /dest
since they will be present in changed files, I didn't see the value of listing the names

Regarding the commit ID, it's also displayed in Commit tab, what would be the value to duplicate it ?

Reply:
> the names of the 3 files in /dest since they will be present in changed files, I didn't see the value of listing the names

Ok
 
> Regarding the commit ID, it's also displayed in Commit tab, what would be the value to duplicate it ?

I'm fine with this